### PR TITLE
[Skeleton Tool] Synfig crashes when we try to edit any other duck other than a bone's

### LINKED ISSUE #1678: [Skeleton Tool] Synfig crashes when we try to edit any other duck other than a bone's
https://youtu.be/mQMvfkzivbI

### PR DESCRIPTION
**Pull request by Aa20475**
Closes #1678

### CODE REVIEW

File: synfig-studio/src/gui/states/state_bone.cpp [hunk 1]
```diff
@@ -982,8 +982,10 @@ StateBone_Context::event_mouse_release_handler(const Smach::event& x)
 			}
 			else{
 				if(duck){
-					_on_signal_change_active_bone(duck->get_value_desc().get_parent_value_node());
-					get_work_area()->set_active_bone_value_node(duck->get_value_desc().get_parent_value_node());
+					if(duck->get_value_desc().is_parent_desc_declared() && ValueNode_Bone::Handle::cast_dynamic(duck->get_value_desc().get_parent_desc().get_value_node())){
```
Comment (rodolforg) [inline synfig-studio/src/gui/states/state_bone.cpp:985]: what about store `duck->get_value_desc().get_parent_desc().get_value_node()` in a variable? :)

Comment (Aa20475) [reply synfig-studio/src/gui/states/state_bone.cpp:985]: Yea okay! 😃

Comment (rodolforg) [reply synfig-studio/src/gui/states/state_bone.cpp:985]: In this  scope I mean. You use it 3 times and it's so long to read lol

Comment (Aa20475) [reply synfig-studio/src/gui/states/state_bone.cpp:985]: Haha yea xD

Comment (morevnaproject) [reply synfig-studio/src/gui/states/state_bone.cpp:985]: I will wait for changes then. ^__^

**Pull request merged**